usr/share/rear/prep/NETFS/GNU/Linux/205_inspect_tar_capabilities.sh

```diff
@@ -0,0 +1,31 @@
+# prep/NETFS/GNU/Linux/205_inspect_tar_capabilities.sh
+# The purpose is to inspect the 'tar --usage' for certain capabilities and if found
+# add these to the BACKUP_PROG_OPTIONS array
+# See also issue https://github.com/rear/rear/issues/1175
+
+# As we will only inspect 'tar' we make one big if block:
+
+if [[ "$(basename $BACKUP_PROG)" = "tar" ]] ; then
+    # Verify extended attributes being present:
+    if tar --usage | grep -q "\-\-xattrs" ; then
```

Comment:
IMHO `tar --usage | grep -- --xattrs` is easier to read (or quote the search term to make it even more visible)

Reply:
@schlomo thanks for the tip.